vendor/bat-native-ledger/src/bat/ledger/internal/wallet/recover.cc

```diff
@@ -32,6 +32,12 @@ Recover::~Recover() {
 void Recover::Start(
     const std::string& pass_phrase,
     ledger::RecoverWalletCallback callback) {
+  if (pass_phrase.empty()) {
```

Comment:
If `pass_phrase` was `empty` before this defensive code was added, could it have caused any side effects when recovering a wallet?